.github/workflows/main.yaml

```diff
@@ -187,7 +187,9 @@ jobs:
           -archivePath AriesBifold.xcarchive \
           -sdk iphoneos \
           -verbose \
-          archive
+          archive \
+          CODE_SIGNING_ALLOWED=NO \
+          CODE_SIGNING_REQUIRED=NO
```

Comment:
Did you mean to include this? If so, should probably be a separate PR I think

Reply:
woops ya I don't know how that got there